lib/utils.sh

```diff
@@ -206,3 +206,35 @@ get_asdf_config_value() {
         get_asdf_config_value_from_file $default_config_path $key
     fi
 }
+
+asdf_repository_url() {
+  echo "https://github.com/asdf-vm/asdf-plugins.git"
+}
```

Comment:
Maybe move this to the top of the file under the `asdf_version` function since it's basically a constant.